src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -18,9 +18,12 @@ namespace Microsoft.AspNetCore.Components.Server.Circuits;
 internal partial class CircuitHost : IAsyncDisposable
 #pragma warning restore CA1852 // Seal internal types
 {
+    public const int TransmitStreamFailedTimeoutSeconds = 10;
```

Comment:
This needs to be configurable via options like other limits

Reply:
We have other timeouts similar to this one (like [here](https://github.com/dotnet/aspnetcore/blob/79a59a5dc16f74ea43b6599fbfb1fcde23005b20/src/Components/Server/src/Circuits/RemoteJSRuntime.cs#L163)) that aren't configurable, and I would consider this to be in the same realm as those. What's the reason we need this to be configurable? Just to make sure it's clear, this is the time the server is willing to wait for the client to acknowledge the stream transmission error before completing the streaming hub invocation.